html_app/model/Treatment.js

```diff
@@ -73,25 +73,34 @@ scb.Treatment = function scb_Treatment(data, context, parent) {
 	Object.defineProperty(self, 'duration', {
 		get : function() {
 			var time = parseFloat(data.duration_value);
-			//var days = Math.floor(time / 86400);
 			var days = Math.floor((time % 604800) / 86400);
-
 			var hours = Math.floor((time % 86400) / 3600);
-			var minutes = Math.round((time % 3600) / 60);
-			
+			var minutes = Math.floor((time % 3600) / 60);
 			var months = Math.floor(time /2592000);
-			
 			var weeks = Math.floor((time % 2592000) / 604800);
-			var now = (time < 60 );
+			var now = (time < 30 );
```

Comment:
My bad. I was completely lost. 

But we still do have very similar code in https://github.com/annagav/starcellbio_html/blob/feature/sutd_assignment2/html_app/model/Treatment.js#L45-L64 and https://github.com/annagav/starcellbio_html/blob/feature/sutd_assignment2/html_app/model/Treatment.js#L45-L64